pkg/containertools/getimagedata_options.go

```diff
@@ -0,0 +1,14 @@
+package containertools
+
+type GetImageDataOptions struct {
+	WorkingDir string
```

Comment:
Nice. You could probably make `OutputDir` an option too (and remove the second parameter of `GetImageData`).

Aside from that, I can't help but think of taking a `Writer` and `Reader` instead of directory strings -- thought that's probably premature optimization.

Reply:
My attempt here was to add just the optional parameters and `OutputDir` is required.